indigo/distribution.yaml

```diff
@@ -10806,6 +10806,23 @@ repositories:
       url: https://github.com/rst-tu-dortmund/teb_local_planner.git
       version: master
     status: developed
+  teleop-twist-joy-xbot-release:
+    doc:
+      type: git
+      url: https://github.com/giupal/teleop_twist_joy_xbot-release.git
```

Comment:
This should point to your source repository instead of the release repo.

Same for the source block below.